frontend/cypress/tests/errorSummary_messageBox.js

```diff
@@ -0,0 +1,32 @@
+describe('error summary', () => {
```

Comment:
please rename this file to `chartView_errorSummary_messageBox.js`, so that all files begin with the major component's name